config/performance_metrics.yml

```
@@ -1,5 +1,5 @@
 performance_metrics:
   organisations: 69
-  published: 182
-  submissions: 278929
+  published: 201
+  submissions: 284367
   last_updated: 2025-07-14
```

@AnneCarr don't forget to update the last_updated date:
```suggestion
  last_updated: 2025-07-21
```